system-addon/test/unit/lib/SectionsManager.test.js

```diff
@@ -226,6 +229,13 @@ describe("SectionsFeed", () => {
     });
   });
   describe("#onAction", () => {
+    it("should bind this.init to SectionsManager.INIT on INIT", () => {
+      sinon.spy(SectionsManager, "once");
+      feed.onAction({type: "INIT"});
+      assert.calledOnce(SectionsManager.once);
+      assert.calledWith(SectionsManager.once, SectionsManager.INIT, feed.init);
+      SectionsManager.once.restore();
```

Comment:
can you add this to an `afterEach()` block? This won't get get hit if one of the assertions throw

Reply:
I realised that we actually don't need that line there at all as SectionsManager is re-decorated with EventEmitter already in afterEach() already